contents/docs/getting-started/install.mdx

```diff
@@ -6,11 +6,8 @@ featuredImage: >-
 hideAnchor: true
 ---
 
-
-> **Tip**: If you have multiple customer-facing products (e.g. a marketing website + iOS app + web app), it's best to install PostHog on them all and [group them in one project](/docs/settings/projects). This way you can track the user across their entire journey.
```

Comment:
nice :)

Reply:
yes, this boxout has been annoying me for a long time.